eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/NewEpoch.hs

```diff
@@ -162,7 +162,7 @@ newEpochTransition = do
       , eNo
       ) <-
     judgmentContext
-  if eNo /= eNoL + 1
+  if eNo /= EpochNo (unEpochNo eNoL + 1)
```

Comment:
I think this is a little bit more type appropriate, despite being a little bit more verbose:
```suggestion
  if eNo /= addEpochInterval eNoL + EpochIntterval 1
```

Reply:
You probably meant 
```haskell
  if eNo /= addEpochInterval eNoL (EpochIntterval 1)
```
I think it's also possible to do
```haskell
  if eNo /= succ eNoL
```